service/dependencies.gradle

```diff
@@ -23,18 +23,21 @@ dependencies {
 
   // Terra deps
   implementation group: "bio.terra", name: "datarepo-client", version: "2.13.0-SNAPSHOT"
-  implementation group: "bio.terra", name:"billing-profile-manager-client", version: "0.1.536-SNAPSHOT"
-  implementation group: "bio.terra", name:"terra-policy-client", version:"1.0.9-SNAPSHOT"
+  implementation group: "bio.terra", name:"billing-profile-manager-client", version: "0.1.611-SNAPSHOT"
```

Comment:
this may not be necessary, but this is the client version of BPM which uses TCL 1.1.38.